### DISCUSSION TITLE
Waring message on chrome developer console

### DISCUSSION THREAD

**Discussion by TechnicalSuneja25** [gatsbyjs/gatsby#31956] [Help]
Hi Team,

Hope you are doing well and safe.

We are working on one project which is based on Gatsby Technology and we are getting the below Waring message on chrome developer console so we need your support on this to check and advise.

After 5-10 sec of the page load , immediately we are getting below warning in the browser console.

 "The resource <URL> was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally."
 
Getting warning message on both local as well as non-live environment .Could not figure out what is the issue 
 
The Gatsby plugin used are as below :
 
 plugins: [
        'gatsby-plugin-react-helmet',
        'gatsby-plugin-sass',
        {
            resolve: 'gatsby-source-filesystem',
            options: {
                name: 'images',
                path: `${__dirname}/src/images`,
            },
        },
        'gatsby-transformer-sharp',
        'gatsby-plugin-sharp',
        {
            resolve: 'gatsby-plugin-manifest',
            options: {
                lang: 'en',
                name: 'gatsby-starter-default',
                short_name: 'starter',
                start_url: '/',
                background_color: '#663399',
                theme_color: '#663399',
                display: 'minimal-ui',
                crossOrigin: `use-credentials`,
                icon: `${__dirname}/src/images/iag-icon.png`,
                localize: [
                    {
                        start_url: '/es/home',
                        lang: 'es',
                        name: 'update me',
                        short_name: 'update me',
                        description: 'update me',
                    },
                ],
            },
        },
       
        {
            resolve: 'gatsby-plugin-env-variables',
            options: {
                whitelist: ['CONTENTFUL_SPACE_ID', 'CONTENTFUL_ACCESS_TOKEN', 'CONTENTFUL_ENVIRONMENT_ID', 'CONTENTFUL_PREVIEW_URL', 'ENABLE_GATSBY_REFRESH_ENDPOINT'],
            },
        },
        {
            resolve: 'gatsby-source-contentful',
            options: contentfulConfig,
        },
        
        `@contentful/gatsby-transformer-contentful-richtext`,
        
        'gatsby-plugin-lodash',
    // this (optional) plugin enables Progressive Web App + Offline functionality
    // To learn more, visit: https://gatsby.dev/offline
    // `gatsby-plugin-offline`,
    ],

Gatsby CLI version: 3.4.0
Gatsby version: 2.32.13
 
Could you please check and help us out on this ?

**Comment by LekoArts** [comment]
The site will work fine (as it's just a warning) but there's a PR that might fix it: https://github.com/gatsbyjs/gatsby/pull/28982
